src/NHibernate.Test/NHSpecificTest/NH3954/EqualsFixture.cs

```diff
@@ -48,9 +49,9 @@ public void TypeInequality()
 		[Test]
 		public void InterfaceEquality()
 		{
-			var entry1 = new ProxyCacheEntry(typeof(Entity1), new[] { typeof(INHibernateProxy), typeof(IProxy) });
```

Comment:
Just wanted to reduce usages of `DynamicProxy.IProxy` as this interface would be dead in a few versions.